core/dovecot/start.py

```diff
@@ -32,6 +33,15 @@ def start_podop():
 for dovecot_file in glob.glob("/conf/*.conf"):
     conf.jinja(dovecot_file, os.environ, os.path.join("/etc/dovecot", os.path.basename(dovecot_file)))
 
+try:
+    os.mkdir("/conf/bin")
+except FileExistsError:
+    pass
+for script_file in glob.glob("/conf/*.script"):
+    out_file = os.path.join("/conf/bin/", os.path.basename(script_file).replace('.script',''))
+    conf.jinja(script_file, os.environ, out_file)
```

Comment:
I personally find this harder to read than plain old 0555.

Reply:
mhh, ack too. Note that python2 != python3 in leading `0` being enough for octal … \*grml\*